src/components/search/SideFacetDropdown.vue

```diff
@@ -233,6 +269,17 @@
           profile: 'facets',
           facet: this.name
         };
+      },
+
+      criteria() {
+        return this.searchFacet.trim().toLowerCase();
+      },
+
+      searchOptions() {
+        if (this.criteria && this.sortedOptions.length === 0) {
+          return 'There are no tags matching your search criteria';
+        }
+        return '';
       }
```

Comment:
Why do we need this for the dropdown form description and don't we use the translated value ( `sideFilters.noOptions`) as is done for the actual shown text?

Reply:
To be honest, I'm not sure if we need this description at all. What would be the use case? 

It's a help text ((https://bootstrap-vue.org/docs/components/form-group#comp-ref-b-form-group-props)) and when it will be displayed currently, it will be somewhere next to the search icon and wraps the search field. If we decide to keep it, we should style it.

Reply:
I'm removing it.